e2e/nomostest/portforwarder/port_forwarder.go

```diff
@@ -31,6 +31,34 @@ import (
 	"kpt.dev/configsync/pkg/kinds"
 )
 
+var errTerminal = errors.New("terminal PortForwarder error")
+
+func isTerminalError(err error) bool {
+	return errors.Is(err, errTerminal)
+}
+
+func newTerminalError(format string, args ...interface{}) error {
+	return errors.Wrapf(errTerminal, format, args...)
+}
+
+// EventType is a type of event sent by PortForwarder
+type EventType string
+
+const (
+	// ErrorEvent indicates a fatal error occurred which caused PortForwarder to exit
+	ErrorEvent EventType = "ERROR"
+	// InitEvent indicates PortForwarder was started and is ready
+	InitEvent EventType = "INIT"
+	// FinishedEvent indicates PortForwarder exited gracefully
+	FinishedEvent EventType = "FINISHED"
```

Comment:
Finished -> Done.

"Done" is conventional go lingo for finished/complete, explicitly including teardown. The pre-teardown convention is "Stopped". e.g. `Stop()` or closing `stopCh` flips the `stopping/stopped` bool, and `doneCh` is closed when teardown is complete.

I don't know if we need a Stopped event here, but we can always add once later if we do.

Reply:
done